A89-backend-service-metric-label.md

```diff
@@ -0,0 +1,131 @@
+A89: Backend Service Metric Label
+----
+* Author(s): [Eric Anderson](https://github.com/ejona86)
+* Approver: @markdroth
+* Status: Ready for Implementation
+* Implemented in: <language, ...>
+* Last updated: 2025-04-01
+* Discussion at: https://groups.google.com/g/grpc-io/c/s4tm26RiMyI
+
+## Abstract
+
+Add a new optional label to per-call metrics containing the "backend service"
+being used for the RPC. When using xDS, the backend service would be the xDS
+cluster name.
+
+## Background
+
+[gRFC A78][] added the `grpc.lb.locality` per-call optional label, which also
+added the infrastructure to support LBs adding optional labels to per-call
+metrics. The optional label can be enabled in the gRPC/OpenTelemetry integration
+API added in [gRFC A79][].
+
+Similar to how locality metrics are useful for analyzing _where_ traffic is
+being routed, the xDS cluster is useful for knowing _to whom_ it is being
+routed. `grpc.target` is generally all that's necessary to know which service is
+receiving traffic, but non-deterministic routing in xDS like weighted clusters,
+aggregate clusters, and cluster specifier plugins mean different clusters (and
+thus potentially different services or service versions) would comingle metrics
+unless the selected cluster is added as a label. It can also be helpful to know
+the selected cluster to confirm that deterministic routing, like path matching,
+is behaving as expected.
+
+### Related Proposals:
```

Comment:
We should probably also reference A66 here, since that's where the per-call metrics are defined.